clrdefinitions.cmake

```diff
@@ -61,6 +61,12 @@ if (CLR_CMAKE_PLATFORM_UNIX)
 
 endif(CLR_CMAKE_PLATFORM_UNIX)
 
+if(CLR_CMAKE_PLATFORM_ALPINE_LINUX)
+  # Alpine Linux doesn't have fixed stack limit, this define disables some stack pointer
```

Comment:
I have written a simple test that basically gets the stack limit and size, then calls alloca to allocate as much as the stack size returned and then writes to the lowest byte of the allocated zone. On Ubuntu, it crashes with SIGSEGV, on Alpine, it passes and I can see in GDB (info proc map) that the stack got extended.